common/server_protocolhandler.cpp

```diff
@@ -281,13 +281,15 @@ void Server_ProtocolHandler::processCommandContainer(const CommandContainer &con
     if (deleted)
         return;
     
-    lastDataReceived = timeRunning;
-    
     ResponseContainer responseContainer(cont.has_cmd_id() ? cont.cmd_id() : -1);
     Response::ResponseCode finalResponseCode;
     
-    if (cont.game_command_size())
+    if (cont.game_command_size()) {
```

Comment:
@Daenyth yes they are different.
Speaking in the lobby is a RoomCommand::ROOM_SAY.
Speaking in-game is a GameCommand::GAME_SAY.